src/lib/components/Flowbite/ThemeContext.tsx

```diff
@@ -6,11 +6,41 @@ import defaultTheme from '../../theme/default';
 export type Mode = string | undefined | 'light' | 'dark';
 
 interface ThemeContextProps {
-  theme: any;
+  theme: FlowbiteTheme;
   mode?: Mode;
   toggleMode?: () => void | null;
 }
 
+export interface FlowbiteTheme {
```

Comment:
THANK YOU! But... may we put it into a separate file? 'Cause, it going to start to be huge. Maybe `ThemeInterface.ts`